test/e2e/webdriver/chrome.js

```diff
@@ -7,14 +7,20 @@ const { ThenableWebDriver } = require('selenium-webdriver'); // eslint-disable-l
  *
  * @type {string}
  */
-const HTTPS_PROXY_HOST = '127.0.0.1:8000';
+const HTTPS_PROXY_HOST = `${
+  process.env.SELENIUM_HTTPS_PROXY || '127.0.0.1:8000'
```

Comment:
might be nice to add these new ENV variables to `.metamaskrc.dist` with descriptions

Reply:
I second this